### DISCUSSION TITLE
EW11 100% Failure on 'send' but receive is OK

### DISCUSSION THREAD

**Discussion by eyadid** [tagyoureit/nodejs-poolController#962] [RS485]
I just finished wiring up my EW11 adapter to an EasyTouch 2.8 controller.
For some reason, i`m getting 100% failure rate on TX, but RX is at ~1%.
The system will receive status correctly, temperature, schedules - all of these show up OK on my web dashboard, but when I try to send any command, eg. update settings, switching from POOL to SPA - the command fails

This suggests that the wiring + UART settings are correct on the EW11, and i`m suspecting i`m missing something on the njspc side (?)

Has anyone experienced this issue ? 

![image](https://github.com/tagyoureit/nodejs-poolController/assets/1462253/1f7e5860-4b91-4b2c-a5f7-7845557fbd05)

**Comment by tagyoureit** [comment]
I'd carefully check (or replace) the wires because there could be a break or bad connection.  It's rare to see comms work in one direction but not the other.

**Reply by eyadid** [reply]
1. The wires are ~1in long, no breaks. Tested continuity and everything is good
2. Iv'e ordered a new EW11 thinking maybe the unit is defective. Still having the same issue
3. When the EasyTouch panel is attached directly to my server (using an RS485 adapter) - everything works OK 

I`m suspecting the latency over WIFI is causing the issues. The buffers on the EW11 are at 1024b. 

Will test over the next few days:
1. Place server next to EW11 and connecting directly it (as AP, not over my home network)
2. Increase buffer sizes (1024 is plenty but maybe it`ll help)
3. Now that I have 2 EW11, i`ll set them up to talk to each other and see if there's an issue there

**Reply by rstrouse** [reply]
Double check the settings on the EW11.  There are a bunch of them and it should be set up as a TCP server and it should be set to the correct baud rate, stop bits, ...etc.  Latency of wifi should not matter so much since the wifi transmission is guaranteed.  I do not have an EW11 and every time I see one it is a matter of going through the settings on the device.
